[IMP] update content of "Down Payment" (Sales V13 - CDU) (#577)
odoo/documentation-user: sales/invoicing/down_payment.rst

Indeed, the buyer shows his will to acquire the product by the down payment and agrees to pay the
rest later, while the seller is obliged to provide the goods by accepting it.

Sometimes a down payment is required for expensive orders or projects. That way, you can protect
yourself and make sure that your customer is serious.

First time you request a down payment
=====================================

When a sales order is confirmed, you will then have the possibility to create an invoice. Invoices
will be automatically created in drafts so that you can review them before validation. To create an
invoice, Odoo Sales will offer you 3 options:

- Regular invoice
- Down payment (percentage)
- Down payment (fixed amount)

In terms of down payment, it can either be a fixed amount or a percentage of the total amount. The
first time you will request a down payment you will be able to put the percentage or the fixed
amount of your choice, select an income account and taxes to precise your down payment. These
settings will be reused for future down payments.

.. image:: media/down_payment_1.png
   :align: center
   :class: img-thumbnail

>    :alt: How can you do a down payment on Odoo Sales?

Reviewer: See my email about alt tags.
For future docs. Don't bother too much for this one :)

For this first example, we will use a 50% amount down payment with a product using an
**ordered quantities** as invoicing policy. Be sure to check out our documentation about invoicing
policies here: :doc:`invoicing_policy` to facilitate your use of this feature. When it comes to
create and view the invoice, you will only have access to a draft invoice mentioning the down
payment (as you can see below).

.. image:: media/down_payment_2.png
   :align: center
   :class: img-thumbnail
   :alt: What is a basic sales flow using down payments on Odoo Sales (1)?

There, you can post the invoice and register the payment of your customer. But, we all know that in
real life this flow does not happen immediately. So, for now, you can return to the sales order.
There, you will have the possibility to see the order as a customer with the **Customer preview**
button or to reach easily the previous draft invoice with the **Invoice** button. In any case, the
down payment will be mentioned on both (sales order and draft invoice).

.. image:: media/down_payment_3.png
   :align: center
   :class: img-thumbnail
   :alt: What is a basic sales flow using down payments on Odoo Sales (2)?

> To complete the flow, if the customer wants to pay the rest of his sales order, you must create

Reviewer: ```suggestion
To complete the flow, when the customer wants to pay the rest of his sales order, you must create
```
they had an agreement :p

another invoice. Once again, you will have the choice to make another down payment or to deduct all
the down payments and so, paying the rest of the invoice as a regular invoice.

.. image:: media/down_payment_4.png
   :align: center
   :class: img-thumbnail
   :alt: What is a basic sales flow using down payments on Odoo Sales (3)?

> This flow is possible also with a down payment taking into account a fixed amount.

Reviewer: ```suggestion
This flow is also possible with a down payment taking into account a fixed amount.
```

.. important::

>    Be careful that if you do a down payment with a product using a **delivered quantities** as

Reviewer: ```suggestion
   Be careful that if you do a down payment with a product using **delivered quantities** as
```

>    invoicing policy, you won’t be able to deduct all the down payments when it comes to invoice your

Reviewer: ```suggestion
   invoicing policy, you won’t be able to deduct all the down payments when it comes to invoicing your
```

   customer. Indeed, you have to deliver a product before creating the final invoice. If nothing has
   been delivered, you will create a **credit note** that will cancel the draft invoice created
   after down payment. To do so, you have to install the **Inventory App** to confirm the delivery

>    or you have to enter manually the delivered quantity on the sales order.

Reviewer: ```suggestion
   customer. Indeed, you have to deliver a product before creating the final invoice. If nothing has
   been delivered, you create a **credit note** that cancels the draft invoice created
   after the down payment. To do so, you have to install the **Inventory App** to confirm the delivery.
   Otherwise, you can enter the delivered quantity manually on the sales order.
```
